src/connection.c

```diff
@@ -75,6 +75,8 @@ static struct connection *conn_create_server(struct context *ctx,
     struct connection *server = server_create(ctx, fd);
     struct conn_info *info = server->info;
     memcpy(&info->addr, addr, sizeof(info->addr));
+    extern const size_t CMD_NUM;
+    info->slow_cmd_counts = cv_calloc(CMD_NUM, sizeof(uint32_t));
```

Comment:
If only server connection should have this field, maybe we should get it out of `struct connection`?